pkg/platform/kube/test/suite.go

```diff
@@ -227,6 +234,20 @@ def handler(context, event):
 	return createFunctionOptions
 }
 
+// InvokeFunction invokes function via HTTP trigger
+func (suite *KubeTestSuite) InvokeFunction(method string, port int, path string, requestBody []byte) {
```

Comment:
for the record, you have `KubectlInvokeFunctionViaCurl` which invoke a non-exposed function via curl pod using function service address

Reply:
@liranbg good to know!